README.md

```diff
@@ -56,6 +56,8 @@ await app.register(fastifyOtelInstrumentation.plugin());
 app.get('/', () => 'hello world')
 // as well as your instance level hooks.
 app.addHook('onError', () => /* do something */)
+// Manually skip telemetry for a specific route
+app.get('/healthcheck', { otel: false }, () => 'Up!')
```

Comment:
We should prefer this interface to benefit the `request.routeOptions.config` getter

```suggestion
app.get('/healthcheck', { config: { otel: false } }, () => 'Up!')
```